scripts/tests/test_validate_docstrings.py

```diff
@@ -1052,6 +1052,11 @@ def test_raises_for_invalid_attribute_name(self, invalid_name):
         with pytest.raises(AttributeError, match=msg):
             validate_docstrings.Docstring(invalid_name)
 
+    @pytest.mark.parametrize('name', ['pandas.Series.str.isdecimal'])
+    def test_encode_content_write_to_file(self, name):
+        docstr = validate_docstrings.Docstring(name).validate_pep8()
```

Comment:
Can you add reference to GH Issue number as comment?

Reply:
do you mean as a comment in my commit or in the actual script?

Reply:
As a comment in the first line of the test. You'll see this in other tests in the code base if you poke around